ecl/sss/v1/approval_requests/results.go

```diff
@@ -6,11 +6,11 @@ import (
 )
 
 type Action struct {
-	Service string `json:"service"`
-	Region  string `json:"region"`
-	APIPath string `json:"api_path"`
-	Method  string `json:"method"`
-	Body    string `json:"body"`
+	Service string                 `json:"service"`
+	Region  string                 `json:"region"`
+	APIPath string                 `json:"api_path"`
+	Method  string                 `json:"method"`
+	Body    map[string]interface{} `json:"body"`
```

Comment:
According to the API reference, the data type of the Body is "String", but it actually contains "Object". (Key = resource name, value = json object of Request Body when creating resource.)